include-refactoring/xsimd/types/xsimd_avx512f_register.hpp

```diff
@@ -3,23 +3,24 @@
 
 #include "./xsimd_generic_arch.hpp"
 
-namespace xsimd {
-
-  struct avx512f : generic {
-    static constexpr bool supported() { return XSIMD_WITH_AVX512F; }
-    static constexpr bool available() { return true; }
-    static constexpr unsigned version() { return generic::version(3, 1, 0); }
-    static constexpr std::size_t alignment() { return 64; }
-  };
+namespace xsimd
+{
+    struct avx512f : generic
+    {
```

Comment:
With this reindenting, you're taking the risk of a difficult merge :-)

Reply:
Plus it makes the review more difficult for me :'(

Reply:
arf sorry for that, I could not resist! I did it only on the register files, which re very small and unlikely to change a lot. I won't do it on other files.